hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -293,6 +295,13 @@ public void setVolumeQuota(String volumeName, long quotaInNamespace,
     HddsClientUtils.verifyResourceName(volumeName);
     verifyCountsQuota(quotaInNamespace);
     verifySpaceQuota(quotaInBytes);
+    // If the volume is old, we need to remind the user on the client side
+    // that it is not recommended to enable quota.
+    OmVolumeArgs omVolumeArgs = ozoneManagerClient.getVolumeInfo(volumeName);
+    if (omVolumeArgs.getQuotaInNamespace() == OLD_QUOTA_DEFAULT) {
+      LOG.warn("Volume: {} is a old volume, usedNamespace may be " +
```

Comment:
I feel like "old" is not descriptive enough. 

Will it better to say that " {} is a volume created before version 1.1.0"? Same for the following warning messages.